client/my-sites/plugins/plugin-upload/index.jsx

```diff
@@ -0,0 +1,38 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { connect } from 'react-redux';
+import page from 'page';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Main from 'components/main';
+import HeaderCake from 'components/header-cake';
+import Upload from 'my-sites/themes/theme-upload';
+import { uploadPlugin } from 'state/plugins/upload/actions';
+
+class PluginUpload extends React.Component {
+	back = () => {
+		page.back();
+	}
+
+	render() {
+		const { translate } = this.props;
+
+		return (
+			<Main>
+				<HeaderCake onClick={ this.back }>{ translate( 'Upload plugin' ) }</HeaderCake>
```

Comment:
Hi! I've found a possible matching string that has already been translated **68** times:
`translate( 'Upload Plugin' )` *ES Score: 15*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).